app/src/main/res/values/strings.xml

```diff
@@ -900,6 +900,7 @@
     <string name="set_img_front">Set Image as Front</string>
     <string name="report_img">Report Image</string>
     <string name="detected_ingredients">Detected Ingredients</string>
+    <string name="set_image_name">"Successfully set the image as "</string>
```

Comment:
```suggestion
    <string name="set_image_name">Successfully set the image as </string>
```

Reply:
@teolemon I had to do that for adding the extra space adter `as`. The double quotes will not be rendered in the app.

Reply:
Translators don't have to question themselves about double quotes @PrajwalM2212

Reply:
@teolemon I did not think about from that perspective but it is fixed now